wsgidav.conf.sample

```diff
@@ -64,6 +64,9 @@ port = 8080
 # with Microsoft Office (default: True)
 add_header_MS_Author_Via = True	
 
+# Add custom headers
+response_headers = [('Access-Control-Allow-Origin', 'http://example.org')]
```

Comment:
Same as above: Should be commented out as example, but not enabled by default and use double quotes